packages/office-ui-fabric-react/src/components/Panel/Panel.tsx

```diff
@@ -238,6 +244,13 @@ export class Panel extends BaseComponent<IPanelProps, IPanelState> implements IP
     }
   };
 
+  // Allow the user to scroll within the panel but not on the body
+  private _allowScrollOnPanel(elt: HTMLDivElement | null): void {
+    if (elt) {
```

Comment:
Per discussion; make sure to clean up event handlers in the 2 cases:

1. component unmounts
2. ref changes to null